### DISCUSSION TITLE
Collection IDs - Tina uses filenames as ID's

### DISCUSSION THREAD

**Discussion by BrookJeynes** [tinacms/tinacms#4445] [Ideas]
CC: @JackDevAU @adamcogan @wicksipedia @bradystroud

Tina uses filenames as ID's. This means that when an item is referenced under a collection, its filename is used. This means that once an items name is changed, the filename will no longer match the title. This also means that you cannot change a filename easily without breaking all references to it

Suggestion 1: Add ID property to collection items and have references use that instead.
Suggestion 2: Have TinaCMS auto update all references

<img width="714" alt="image" src="https://github.com/tinacms/tinacms/assets/25432120/0b542005-34b0-4c86-ad68-d8f415a00ce4">

**Figure: Item referenced by name in a collection**

**Comment by ussiemer** [comment]
Definitely a big problem for non-technical users.
